modules/oe_content_person/oe_content_person.module

```diff
@@ -10,6 +10,7 @@ declare(strict_types = 1);
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\Core\Render\Element;
+use Drupal\field\Entity\FieldConfig;
```

Comment:
This is not used, we can remove it:

![image](https://user-images.githubusercontent.com/153362/117136828-35493300-ada9-11eb-94fd-caeca0f10804.png)